### PR TITLE
GLT-3043: correct MinION sequencing container models

### PR DESCRIPTION
**Pull request by djcooke**

### CODE REVIEW

File: scanner/src/main/java/ca/on/oicr/gsi/runscanner/scanner/processor/MinionProcessor.java [hunk 1]
```diff
@@ -48,8 +48,10 @@ protected void additionalProcess(OxfordNanoporeNotificationDto onnd, IHDF5Reader
     if (reader.hasAttribute(trackingId, "device_id")) {
       onnd.setSequencerName(reader.string().getAttr(trackingId, "device_id"));
     }
-    if (reader.hasAttribute(contextTags, "sequencing_kit")) {
-      onnd.setContainerModel(reader.string().getAttr(contextTags, "sequencing_kit"));
+    if (reader.hasAttribute(contextTags, "flow_cell_product_code")) {
```
Comment (avarsava) [inline scanner/src/main/java/ca/on/oicr/gsi/runscanner/scanner/processor/MinionProcessor.java]: PromethionProcessor has the exact same code, could this be moved to the process() in BaseOxfordNanoporeProcessor?

Comment (djcooke) [reply scanner/src/main/java/ca/on/oicr/gsi/runscanner/scanner/processor/MinionProcessor.java]: Moved

File: scanner/src/main/java/ca/on/oicr/gsi/runscanner/scanner/processor/MinionProcessor.java [hunk 2]
```diff
@@ -48,9 +48,6 @@ protected void additionalProcess(OxfordNanoporeNotificationDto onnd, IHDF5Reader
     if (reader.hasAttribute(trackingId, "device_id")) {
       onnd.setSequencerName(reader.string().getAttr(trackingId, "device_id"));
     }
-    if (reader.hasAttribute(contextTags, "sequencing_kit")) {
-      onnd.setContainerModel(reader.string().getAttr(contextTags, "sequencing_kit"));
```
Comment (avarsava) [inline scanner/src/main/java/ca/on/oicr/gsi/runscanner/scanner/processor/MinionProcessor.java]: Will getting rid of the check for sequencing_kit as a last resort break older fast5s, or was it never correct in the first place?

Comment (djcooke) [reply scanner/src/main/java/ca/on/oicr/gsi/runscanner/scanner/processor/MinionProcessor.java]: It was never correct. `sequencing_kit` is the sequencing kit, not the flow cell model. See JIRA ticket for more details

**Pull request merged**